showcase-v1alpha2/protos/google/showcase/v1alpha2/echo.proto

```diff
@@ -0,0 +1,177 @@
+// Copyright 2018 Google LLC
```

Comment:
How much of a burden would it be to write this sample on top of the v0.0.8 showcase protos. I made some breaking changes to make the pagination method more idiomatic and changed wait to be a LRO.

Reply:
I will update this PR with these changes (hopefully today). It may change the contents of the request/response object, but won't change the TypeScript stuff, so it's still OK to review it in this version :)